File: Kyu7/disemvowel.js
function disemvowel(str) {
  const vowels = ["a", "e", "i", "o", "u"];
  for (let i = 0; i < vowels.length; i++) {
    str = str.split(vowels[i]).join("");
    str = str.split(vowels[i].toUpperCase()).join("");
  }
  console.log(str);
  return str;
}

File: Kyu7/disemvowel.py
# def disemvowel(string_):
#   newString = ""
#   for char in string_:
#     if char.lower() not in "aeiou":
#       newString += char
#   return newString

def disemvowel(string_):
  return "".join(char for char in string_ if char.lower() not in "aeiou")

print(disemvowel("This website is for losers LOL!")) # "Ths wbst s fr lsrs LL!"